recipe/bld.bat

```diff
@@ -1,12 +1,21 @@
-REM Microsoft Visual Studio Project files from https://github.com/kmanley/snappy-msvc
-copy %RECIPE_DIR%\snappy.sln %SRC_DIR%
-copy %RECIPE_DIR%\snappy.vcproj %SRC_DIR%
+setlocal EnableDelayedExpansion
 
-set SLN_FILE=snappy.sln
-set SLN_CFG=Release
-if "%ARCH%"=="32" (set SLN_PLAT=Win32) else (set SLN_PLAT=x64)
+:: Make a build folder and change to it.
+mkdir build
+cd build
 
-REM Build step
-devenv "%SLN_FILE%" /Build "%SLN_CFG%|%SLN_PLAT%"
+:: Configure using the CMakeFiles
+%LIBRARY_BIN%\cmake -G "NMake Makefiles" ^
```

Comment:
Sure. This was based on the example in the docs, but I'll remove it and send a cleanup PR to the docs because it's kind of silly.